spring-data-sample/README.md

```diff
@@ -261,7 +261,7 @@ $ ./gradlew run --args="PlaceOrder 1 3:1,4:1"
 
 ## Cleanup
 
-To stop Cassandra and shut down the sample application, run the following command:
+To stop Cassandra, run the following command:
```

Comment:
`docker-compose` manages only schema-loader and Cassandra not the application in this sample

Reply:
Understood. LGTM!